ESP32 temperature-measurement-app fails to build

I can get the all-clusters ESP32 example to build and run, however the temperature-measurement-app fails to build.
 
I have followed Instructions here: https://github.com/project-chip/connectedhomeip/blob/master/examples/temperature-measurement-app/esp32/README.md
 
ESP-IDF release/v4.2
CHIP commit b40ed7a710fca26c1df013c2132e6e073e566ff0
 
This built fine about two weeks ago. 

af-event.cpp:90: undefined reference to `emberAfPluginTemperatureMeasurementServerReadEventControl'

 ```
/home/wpeacock/.espressif/tools/xtensa-esp32-elf/esp-2020r3-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(af-event.cpp.obj):/home/wpeacock/chip/connectedhomeip/examples/temperature-measurement-app/esp32/build/../third_party/connectedhomeip/src/app/util/af-event.cpp:90: undefined reference to `emberAfPluginTemperatureMeasurementServerReadEventControl'
/home/wpeacock/.espressif/tools/xtensa-esp32-elf/esp-2020r3-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(af-event.cpp.obj):(.data.emAfEvents+0x4): undefined reference to `emberAfPluginTemperatureMeasurementServerReadEventHandler()'
collect2: error: ld returned 1 exit status
```


Regards
Wayne

--- a/examples/temperature-measurement-app/esp32/main/gen/af-gen-event.h
+++ b/examples/temperature-measurement-app/esp32/main/gen/af-gen-event.h
@@ -22,15 +22,4 @@
 #ifndef __AF_GEN_EVENT__
 #define __AF_GEN_EVENT__
 
-// Code used to configure the cluster event mechanism
-#define EMBER_AF_GENERATED_EVENT_CODE                                                                                              \
-    extern EmberEventControl emberAfPluginTemperatureMeasurementServerReadEventControl;                                            \
-    extern void emberAfPluginTemperatureMeasurementServerReadEventHandler(void);
-
-// EmberEventData structs used to populate the EmberEventData table
-#define EMBER_AF_GENERATED_EVENTS                                                                                                  \
-    { &emberAfPluginTemperatureMeasurementServerReadEventControl, emberAfPluginTemperatureMeasurementServerReadEventHandler },
-
-#define EMBER_AF_GENERATED_EVENT_STRINGS "Temperature Measurement Server Cluster Plugin Read",
-
 #endif // __AF_GEN_EVENT__

